continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -44,7 +44,8 @@ outputs:
       script: >
         python -m pip install . -vv --no-deps
         --install-option="--with-cython=profile"  # [cython_enabled]
-      track_features: {{ "cythonized-scheduler" if cython_enabled else "" }}
+      track_features: "[cythonized-scheduler]"    # [cython_enabled]
+      track_features: ""                          # [not cython_enabled]
```

Comment:
Is this needed or can this be dropped since no features should be tracked in this case?

Reply:
Good point - can remove the case for `not cython_enabled`

```suggestion
```